spec/requests/api/api_responds_with_a_collection_of_product.rb

```diff
@@ -0,0 +1,37 @@
+
+RSpec.describe 'GET /api/products' do
+  describe 'successfully' do
+    let!(:products) {3.times { create(:product)}}
+    before do
+      get '/api/products'
+    end
+
+    it 'is expected to return a 200 response status' do
+      expect(response).to have_http_status 200
+    end
+       
+    it 'is expected to return all products' do
+      expect(JSON.parse(response.body)['products'].count).to eq 3
```

Comment:
Now that you have the `response_json` helper in /spec/support you should change the line to look as follows:
```js
expect(response_json['products'].count).to eq 3
```